### DISCUSSION TITLE
ADD midi control over parameters like: start-end point samples or midi sequences. Pitch and tempo.

### DISCUSSION THREAD

**Discussion by Lockheed22** [helgoboss/helgobox#1262] [Ideas]
Adding the ability to control the start and end points by assigning a controller (knob or fader) to it.
Also for pitch and tempo.
Now it’s only possible to control them by mouse.
So playtime could function as a sampler like mpc style.
Also instead of choosing points by bars maybe choosing by time would be a great option especially for audio clips.

**Comment by RibbleFripFnibbulator** [comment]
I looked for a way to do this. Need to be free from the mouse for those controls!

**Comment by helgoboss** [comment]
Duplicate of #978.
